db/migrate/20170331170222_change_index_type.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeIndexType < ActiveRecord::Migration
+  def up
+    change_column :readings, :id, "bigint NOT NULL AUTO_INCREMENT"
+  end
+
+  def down
```

Comment:
Can you name this migration a bit more descriptively, e.g. change_readings_id_type?